framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -191,6 +229,79 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		await setChainConnectorInfo(this._chainConnectorPluginDB, chainConnectorInfo);
 	}
 
+	public async calculateCCUParams(
+		sendingChainID: Buffer,
+		certificate: Certificate,
+		newCertificateThreshold: bigint,
+	): Promise<CrossChainUpdateTransactionParams | undefined> {
+		let activeBFTValidatorsUpdate: BFTValidator[];
+		let activeValidatorsUpdate: ActiveValidator[] = [];
+		let certificateThreshold = newCertificateThreshold;
+
+		const blockHeader = this._chainConnectorState.blockHeaders.find(
+			header => header.height === certificate.height,
+		)!;
+
+		const chainAccount = await this._mainchainAPIClient.invoke<ChainAccount>(
+			'interoperability_getChainAccount',
+			{ chainID: sendingChainID },
+		);
+
+		const certificateBytes = codec.encode(certificateSchema, certificate);
+
+		const certificateValidationResult = await this._validateCertificate(
+			certificateBytes,
+			certificate,
+			blockHeader,
+			chainAccount,
+		);
+		if (!certificateValidationResult.status) {
+			this.logger.error(certificateValidationResult, 'Certificate validation failed');
+
+			return undefined;
```

Comment:
I think only `return` will suffice
```suggestion
			return;
```